forge-gui/res/editions/Warhammer 40,000 Commander.txt

```diff
@@ -15,325 +15,165 @@ ScryfallCode=40K
-130★ R Lucius the Eternal @Evan Shipard
 131 R Magnus the Red @Wonchun Choi
-131★ R Magnus the Red @Wonchun Choi
 132 R Malanthrope @Ørjan Ruttenborg Svendsen
-132★ R Malanthrope @Ørjan Ruttenborg Svendsen
 133 R Mawloc @Mathias Kollros
-133★ R Mawloc @Mathias Kollros
 134 R Mutalith Vortex Beast @Kekai Kotaki
-134★ R Mutalith Vortex Beast @Kekai Kotaki
 135 R Neyam Shai Murad @Fajareka Setiawan
-135★ R Neyam Shai Murad @Fajareka Setiawan
 136 R Pink Horror @Games Workshop
-136★ R Pink Horror @Games Workshop
 137 U Primaris Chaplain @Diego Gisbert
-137★ U Primaris Chaplain @Diego Gisbert
 138 R Ravener @Xavier Ribeiro
-138★ R Ravener @Xavier Ribeiro
 139 R The Ruinous Powers @Lie Setiawan
-139★ R The Ruinous Powers @Lie Setiawan
 140 R Shadow in the Warp @Kim Sokol
-140★ R Shadow in the Warp @Kim Sokol
 141 R Sister Hospitaller @Games Workshop
-141★ R Sister Hospitaller @Games Workshop
 142 R Sister Repentia @Seseon Yoon
-142★ R Sister Repentia @Seseon Yoon
 143 U Trygon Prime @Mathias Kollros
-143★ U Trygon Prime @Mathias Kollros
 144 R Tyranid Harridan @Billy Christian
-144★ R Tyranid Harridan @Billy Christian
 145 R Tyranid Prime @Mathias Kollros
-145★ R Tyranid Prime @Mathias Kollros
 146 R Tzaangor Shaman @Wisnu Tan
-146★ R Tzaangor Shaman @Wisnu Tan
 147 U Venomthrope @Antonio José Manzanedo
-147★ U Venomthrope @Antonio José Manzanedo
 148 R Winged Hive Tyrant @Antonio José Manzanedo
-148★ R Winged Hive Tyrant @Antonio José Manzanedo
 149 R Zoanthrope @Billy Christian
-149★ R Zoanthrope @Billy Christian
 150 R Canoptek Scarab Swarm @Alexey Kruglov
-150★ R Canoptek Scarab Swarm @Alexey Kruglov
 151 R Canoptek Spyder @Wisnu Tan
-151★ R Canoptek Spyder @Wisnu Tan
 152 R Canoptek Tomb Sentinel @Ivan Dedov
-152★ R Canoptek Tomb Sentinel @Ivan Dedov
 153 R Canoptek Wraith @Hector Ortiz
-153★ R Canoptek Wraith @Hector Ortiz
 154 R Convergence of Dominion @David Álvarez
-154★ R Convergence of Dominion @David Álvarez
 155 R Cryptothrall @Henry Peters
-155★ R Cryptothrall @Henry Peters
 156 R Ghost Ark @Alexey Kruglov
-156★ R Ghost Ark @Alexey Kruglov
 157 R The Golden Throne @Games Workshop
-157★ R The Golden Throne @Games Workshop
 158 U Goliath Truck @Diego Gisbert
-158★ U Goliath Truck @Diego Gisbert
 159 R Inquisitorial Rosette @Álvaro Calvo Escudero
-159★ R Inquisitorial Rosette @Álvaro Calvo Escudero
 160 R Knight Paladin @Games Workshop
-160★ R Knight Paladin @Games Workshop
 161 R Necron Monolith @Anthony Devine
-161★ R Necron Monolith @Anthony Devine
 162 U Night Scythe @Calder Moore
-162★ U Night Scythe @Calder Moore
 163 R Reaver Titan @Philip Helliwell
-163★ R Reaver Titan @Philip Helliwell
 164 R Redemptor Dreadnought @Philip Helliwell
-164★ R Redemptor Dreadnought @Philip Helliwell
 165 R Resurrection Orb @Hector Ortiz
-165★ R Resurrection Orb @Hector Ortiz
-166 R Sceptre of Eternal Glory @Michał Miłkowski
-166★ R Sceptre of Eternal Glory @Michał Miłkowski
+166 R Sceptre of Eternal Glory @Micha Mi?kowski
```

Comment:
Can you revert card 166 as the artist shouldn't be changed (your edit also contains a ?)